utils/global-state-update-gen/src/validators/validators_manager.rs

```diff
@@ -1,31 +1,40 @@
 use std::{
     cmp::Ordering,
-    collections::{BTreeMap, BTreeSet},
+    collections::{btree_map::Entry, BTreeMap, BTreeSet},
 };
 
 use casper_engine_test_support::LmdbWasmTestBuilder;
 use casper_types::{
+    account::Account,
     system::{
         auction::{
             Bid, Bids, SeigniorageRecipient, SeigniorageRecipientsSnapshot,
             SEIGNIORAGE_RECIPIENTS_SNAPSHOT_KEY,
         },
         mint::TOTAL_SUPPLY_KEY,
     },
-    AsymmetricType, CLValue, EraId, Key, PublicKey, StoredValue, U512,
+    CLValue, EraId, Key, PublicKey, StoredValue, U512,
 };
 
 use crate::utils::{hash_from_str, validators_diff, StateTracker, ValidatorsDiff};
 
+use super::ValidatorConfig;
+
 /// A struct tracking all the changes to the state made during the course of the upgrade.
 pub struct ValidatorsUpdateManager {
+    validators: BTreeMap<PublicKey, ValidatorConfig>,
     builder: LmdbWasmTestBuilder,
     state_tracker: StateTracker,
+    accounts: BTreeMap<PublicKey, Account>,
```

Comment:
This whole struct is confusing to me:
* the comment on the struct doesn't mention validators.
* the name includes _validators_
* there is `validators` entry AND `accounts` entry - are those _any_ accounts or only _validators_'?

Reply:
Well, this became kind of a mess. Initially the `validators` subcommand was only able to modify the validator set, and now that the possibility of affecting accounts with 0 stake is being added, it can also affect non-validators. So this struct is basically all the state associated with the "validators update", which is no longer quite just a validators update... :grimacing: 

I guess I should rename some stuff here.

Reply:
Maybe `AccountsUpdate` then? It allows for updating stake and balances after all.